cub/test/catch2_test_device_transform.cu

```diff
@@ -544,3 +560,101 @@ C2H_TEST("DeviceTransform::Transform aligned_base_ptr", "[device][device_transfo
   STATIC_REQUIRE(::cuda::std::is_constructible_v<kernel_arg>);
   STATIC_REQUIRE(::cuda::std::is_copy_constructible_v<kernel_arg>);
 }
+
+C2H_TEST("DeviceTransform::Transform vectorized output bug", "[device][device_transform]")
+{
+  using thrust::placeholders::_1;
+
+  int num_items = std::numeric_limits<std::uint16_t>::max() - 1;
+  c2h::device_vector<std::uint16_t> input(num_items);
+  c2h::device_vector<std::uint16_t> output(num_items);
+  thrust::sequence(input.begin(), input.end());
+
+  // auto out_it = output.begin(); // works
```

Comment:
suggestion: this comment (and one in the test below) is likely a copy-paste, let's remove it. Both this and one below should work.

Reply:
I retained the comment because it was in the test you provided. I removed it now.